### DISCUSSION TITLE
hreflang support

### DISCUSSION THREAD

**Discussion by robrechtme** [i18next/next-i18next#1809] [Ideas]
Hi 👋🏻 

Adding `hreflang` tags to your pages head improves SEO. I was reading [this](https://nextjs.org/docs/advanced-features/i18n-routing#search-engine-optimization) in the NextJS docs:

> Next.js doesn't know about variants of a page so it's up to you to add the `hreflang` meta tags using [`next/head`](https://nextjs.org/docs/api-reference/next/head). You can learn more about hreflang in the [Google Webmasters documentation](https://support.google.com/webmasters/answer/189077).

When using `next-i18next`, every page will be available in all locales, so we could add the `hreflang` tags to `appWithTranslation`, no? If you're interested I can make a PR.

**Comment by isaachinman** [comment]
Due to limitations within NextJs, this is a pretty tricky issue and probably belongs more to a project like `next-sitemap` than `next-i18next`.

As they say themselves:

> Next.js doesn't know about variants of a page

Feel free to elaborate on how you planned to solve this – let's definitely discuss in detail before any potential work begins.
